pypeit/calibrations.py

```diff
@@ -209,6 +210,44 @@ def __init__(self, fitstbl, par, spectrograph, caldir, qadir=None,
         self.success = False
         self.failed_step = None
 
+    def check_calibrations(self, file_list, check_lamps=True):
```

Comment:
Can you make it more clear what this routine does? It does not return anything and it does not appear to crash out if the calibrations are not okay. It appears to only print warnings to the screen. Can you please make that clear in the docs.